devpi-nginx.env

```diff
@@ -0,0 +1,41 @@
+# DEVPI_SERVERDIR is the directory where devpi-server store data.
+# /devpidata is a data volume created via the Dockerfile (see https://docs.docker.com/userguide/dockervolumes/)
+# If you change this variable, you may want to set up you own data volume.
+DEVPI_SERVERDIR=/mnt
+
+# DEVPI_CLIENTPASSWORD is the default password for the "root" user
+DEVPI_CLIENTPASSWORD=
+
+# NGINX_IP is the IP on which NGINX listens to connnections.
+# 0.0.0.0 means it listens on every interface, which may no be what you want.
+NGINX_IP=0.0.0.0
+
+# NGINX_PORT is the port on which NGINX listens to connections.
+# 3141 port is already exposed via the Dockerfile but you can change it as long as you specify your port via 
+# the -p option when running your container.
+NGINX_PORT=3141
+
+# if NGINX_ENABLETLS is set to true, TLS will be enabled for requests made on
+# ${NGINX_IP}:${NGINX_TLSPORT}
+NGINX_ENABLETLS=true
+NGINX_TLSPORT=3142
+
+# this docker image comes with a builtin self-signed certificate but if you
+# chose to enable TLS you should specify the path to your own certificate and
+# its associated key via ${NGINX_PATH}/${NGINX_CERT} and
+# ${NGINX_PATH}/${NGINX_TLSKEY} respectively
+#
+# It is recommended to use NGINX_PATH=/opt/nginx/certs/custom to put your own
+# certificates.
+NGINX_TLSPATH=/opt/nginx/certs/default
+NGINX_TLSCERT=devpi.crt
+NGINX_TLSKEY=devpi.key
+
+# Change following variables if you know what your doing
+# Look at run.sh to know what they do
+DEVPI_SERVERIP=127.0.0.1
+DEVPI_SERVERPORT=3143
+DEVPI_CLIENTDIR=/tmp/devpi-client
+
+# For future use
+DEVPI_LDAPCONFILE=
```

Comment:
Please remove all references to LDAPCONFILE, if we eventually add it will come in its own pull request.